docs/user-guide/deployment-scenarios.md

```diff
@@ -0,0 +1,51 @@
+# Deployment Scenarios
+
+The StarterKit support a wide variety of deployments, and this article will
+highlight some of them.
+
+- [Keep data on-premise](#keep-data-on-premise)
+- [Redundancy with multiple concentrators and gateways](#redundancy-with-multiple-concentrators-and-gateways)
+- [Deployment on Windows Server](#deployment-on-windows-server)
+- [Deployment of IoTEdge in the cloud](#deployment-of-iotedge-in-the-cloud)
+
+Other scenarios are supported, and combinations of the below scenario are possible.
+
+## Keep data on-premise
+
+It is possible to deploy Azure IoT Edge on your own hardware, and keep all data
+in your local network or infrastructure. For example, by using a custom local
+forwarding module, it is possible to route the data to a local queue or message
+bus. In this case, the sensor data will never leave the on-premise network, and
+connection to Azure will only be used for managing the devices, or handling the
```

Comment:
It's not clear to me what "Management flow" means. I think to be more accurate we would need to differentiate between how the user defines devices (which is how I would understand "management"), and which dependency the starter kit has "internally". I think the Facade function has more functionality than simply managing devices and dedup, it's there to fetch credentials, we use it for firmware upgrades etc.

Reply:
By "management flow" I mean exactly that: updating devices, handling dedup, etc... It is used to differentiate from actual "signal or telemetry data".

Reply:
@bastbu I updated the wording a little bit, does that make sense?

Reply:
Hm, I'm still not sure if it makes sense that the Facade is only there to manage devices and for deduplication. But then again, it's such a detail, not worth changing. Will resolve the comment.